### DISCUSSION TITLE
Bedrock usage tokens

### DISCUSSION THREAD

**Discussion by rjesh-git** [awslabs/multi-agent-orchestrator#180] [Q&A]
Is there anyway we can use the bedrock usage tokens, part of `AgentResponse` so that it can be added to as metrics to observability platform.

https://docs.aws.amazon.com/bedrock/latest/APIReference/API_runtime_TokenUsage.html#bedrock-Type-runtime_TokenUsage-inputTokens

**Comment by cornelcroi** [comment]
We plan to add a feature to have all kinds of metrics available, which will be also customizable and flexible. 
Have a look at the discussion in here https://github.com/awslabs/multi-agent-orchestrator/issues/152.
